packages/plugin-ext-vscode/src/node/plugin-vscode-backend-module.ts

```diff
@@ -21,6 +21,8 @@ import { PluginVsCodeDirectoryHandler } from './plugin-vscode-directory-handler'
 import { VsCodePluginScanner } from './scanner-vscode';
 import { VsCodePluginDeployerResolver } from './plugin-vscode-resolver';
 
+process.env['VSCODE_NLS_CONFIG'] = JSON.stringify({ locale: 'en', availableLanguages: {} });
```

Comment:
I think this needs at least a comment.

Reply:
It actually is the bogus place, should be installed on env of the hosted process. I will move it and add a comment

Reply:
@svenefftinge  moved to the plugin host process: https://github.com/theia-ide/theia/pull/4020/files#diff-66cd2825aa616ad9a26b1f62e1b21e5b
and added the comment